doc/authoring_command_modules/example_module_template/azure/cli/command_modules/example/custom.py

```diff
@@ -3,16 +3,19 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import azure.cli.core.azlogging as azlogging
+from knack.log import get_logger
+from knack.util import CLIError
 
 
-logger = azlogging.get_az_logger(__name__)
+logger = get_logger(__name__)
 
 
-def example_custom(example_param=None):
-    result = {'example_param': example_param}
-    return result
+def create_example(cmd, example_name, resource_group_name, location=None):
+    from azure.mgmt.example.models import Example
+    from azure.cli.command_modules.example._client_factory import cf_example
+    client = cf_example(cmd.cli_ctx)
 
+    # TODO: custom or non-trivial stuff here
```

Comment:
Looks like we are going to merge with this comment still there.
Then maybe a nicer message like 'Add more examples of custom or non-trivial commands'?

Reply:
The intent was "here's where you do whatever you want..." I'll reword it.